Entity/Entities/Bird.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Entities.Interfaces;
+
+namespace Entities.Entities;
+
+public class Bird : IEntity, ICreature, IFlyable
+{
+    public Bird(int id, string name, double maxFlyingHeight)
+    {
+        Id = id;
+        Name = name;
+        MaxFlyingHeight = maxFlyingHeight;
+    }
+
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public double MaxFlyingHeight { get; set; }
+    
+    public override string ToString()
```

Comment:
same. common logic of ToString() method from one class to another

Reply:
fixed